engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -78,57 +76,37 @@
 import static org.lwjgl.opengl.GL11.glPushMatrix;
 
 /**
- * The world of Terasology. At its most basic the world contains chunks (consisting of a fixed amount of blocks)
- * and the player.
- * <p/>
- * The world is randomly generated by using a bunch of Perlin noise generators initialized
- * with a favored seed value.
- *
  * @author Benjamin Glatzel <benjamin.glatzel@me.com>
  */
 public final class WorldRendererLwjgl implements WorldRenderer {
-    public static final int VERTICAL_SEGMENTS = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
 
     private static final int SHADOW_FRUSTUM_BOUNDS = 500;
 
-    /* WORLD PROVIDER */
+    private final int verticalMeshSegments = CoreRegistry.get(Config.class).getSystem().getVerticalChunkMeshSegments();
+
+    private final Skysphere skysphere;
     private final WorldProvider worldProvider;
+    private final RenderableWorld renderableWorld;
 
-    /* PLAYER */
     private LocalPlayer player;
 
-    /* CAMERA */
-    private Camera activeCamera;
+    private final Camera activeCamera;
```

Comment:
Added note: WorldRenderer has a getActiveCamera() method. This is used in many, many other files and for the time being I'd rather leave it as it is rather than rename it getPlayerCamera(). Hopefully in one of the next PRs I'll be able to rename the camera in the code back to activeCamera anyway, by making the code capable of switching cameras at runtime.